resources/config/services/services.yml

```diff
@@ -38,7 +38,6 @@ services:
     bind:
       $clientId: '%opencfpcentral.clientId%'
       $clientSecret: '%opencfpcentral.clientSecret%'
-      $httpClient: '@http_client'
```

Comment:
We can safely remove the binding when we use an actual class (or interface) name for the service definition, as then the auto-wiring mechanism will use the service that corresponds to that class (or interface) name.